### DISCUSSION TITLE
[Feature Request] First/Last Page controls in Pagination component

### DISCUSSION THREAD

**Discussion by WorldOfMaze** [heroui-inc/heroui#1898] [Feature requests]
### Is your feature request related to a problem? Please describe.

Not a problem.

### Describe the solution you'd like

Add _First Page_ `<<` and _Last Page_ `>>` buttons to the Pagination control.

When the _First Page_ button is clicked, the `onChange` callback would be called with a value of `1`.

When the _Last Page_ button is clicked, the `onChange` callback would be called with a value of `total`.

### Describe alternatives you've considered

To implement the behavior I needed, I wrapped the Pagination control with two buttons styled to match the buttons in the pagination control:

```
          <Button
            isIconOnly
            className="min-w-9 box-border h-9 w-9 rounded-medium bg-default-100 text-small text-default-foreground outline-none active:bg-default-300"
            onClick={() => setPage(1)}
          >
            {"<<"}
          </Button>

          <Pagination .../>

          <Button
            isIconOnly
            className="min-w-9 box-border h-9 w-9 rounded-medium bg-default-100 text-small text-default-foreground outline-none active:bg-default-300"
            onClick={() => setPage(pages)}
          >
            {">>"}
          </Button>
```

**Comment by KatuGT** [comment]
I´ve looking for this, I would love to have this functionality
